di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/service/ConfigService.java

```diff
@@ -21,11 +22,14 @@ public class ConfigService {
     private static final Gson GSON = new Gson();
     private static final SsmClient SSM_CLIENT = getSsmClient();
     private static final String CLIENT_CONFIG_BASE_PATH = "/stubs/credential-issuer-stub-clients";
+    private static final String API_KEY_PATH = "/stubs/credential-issuer-stub-api-key";
```

Comment:
In the spirit of moving to a standardised naming convention, could this be `/stubs/credentialIssuerStubApiKey`? Or even `/stubs/cri/apiKey`?

Reply:
Sure